conf/httpd.conf.d/httpd.admin

```diff
@@ -103,6 +103,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
 $User = "pf";
 $Group = "pf";
 
+$PerlChildInitHandler = "pf::WebAPI::ChildInitHandler";
```

Comment:
Also PerlChildInitHandler pf::WebAPI::InitHandler::child_init was removed from httpd.admin.conf
It was needed because of the non-random password generation